packages/boxel/addon/components/boxel/action-container/section/title/index.gts

```diff
@@ -0,0 +1,44 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import cn from '@cardstack/boxel/helpers/cn';
+import cssUrl from "@cardstack/boxel/helpers/css-url";
+import or from 'ember-truth-helpers/helpers/or';
+import { svgJar } from '@cardstack/boxel/utils/svg-jar';
+
+import '@cardstack/boxel/styles/global.css';
+import './index.css';
+
+
+export interface Signature {
+  Element: HTMLElement;
+  Args: {
+    dataTestId?: string;
+    title?: string;
+    icon?: string;
+    imgUrl?: string;
```

Comment:
Should there be a usage example that shows `icon` and `imgUrl`?

Reply:
Good idea. Added.